src/app/pages/dashboard/job-result/job-actions-menu/job-actions-menu.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, Input } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { JobService } from '../../../../@core/services/job.service';
+import { ToastService } from 'src/app/@core/services/toast.service';
+import { Job } from '../../../../@core/models/models';
+
+@Component({
+  selector: 'job-actions-menu',
+  templateUrl: './job-actions-menu.component.html',
+  styleUrls: ['./job-actions-menu.component.scss'],
+})
+export class JobActionsMenuComponent {
+  @Input() job: Job;
+
+  // Job ID whose result is being displayed
+  public jobId: number;
+  // interval var
+  private pollInterval: any;
+  // RxJS Subscription
+  private sub: Subscription;
+  // if true, shows error template
+  public isError: boolean = false;
```

Comment:
Lines 16-23 can be removed.

Reply:
The button functions use `jobId`. Will that be passed from the parent component as a prop then?

Reply:
Use `job.id`